src/electron/lib/Blix.ts

```diff
@@ -60,14 +60,8 @@ export class Blix {
     // this._aiManager = new AiManager(mainWindow);
     this._projectManager = new ProjectManager(mainWindow);
 
-    // Add subscribers
```

Comment:
The graph IPC subscriber is kind of an special case subscriber. Originally the graph subscriber was created in `GraphApi.ts`. I don't think it belongs here though because the MainApis is only there to facilitate communication invoked from the frontend whereas the GraphSubscriber sends the new state to the frontend through the Window/Client API channels. So, for now I think when Blix instantiates it is best to also instantiate all the special subscribers then and there.